system_test/helpers/aest_nodes.erl

```diff
@@ -53,15 +53,24 @@
 
 %% AWK script to keep only error, critical, alert and emergency log lines with
 %% all the extra lines following the log lines.
+%% FIXME: Temporarily ignore dispatch_worker errors, remove when fixed.
```

Comment:
Could you please ensure a ticket tracking this removal exists in Pivotal cc @sennui ? It does not need to be included in the code.

Reply:
done.